package.json

```diff
@@ -32,6 +32,7 @@
     "prism-react-renderer": "^1.1.1",
     "react": "^17.0.1",
     "react-dom": "^17.0.1",
+    "react-feather": "^2.0.9",
```

Comment:
Does this mean that the `documentation` project will not be using the icons from the DS project?

Reply:
This is just temporary until we sort out the icons in the component lib, there are likely to be a fair few changes

Reply:
oh ok then.